### PR TITLE
Unattended fix dev in testing

### PR DESCRIPTION
**Pull request by lastranikos**
|Related issue|
|---|
|https://github.com/wazuh/wazuh-packages/issues/1036|

## Description

- Fix Creating certificates case
- Merge branch 'unify-unatteded-dev' into 1036-unattended-fix-dev-in-testing

### CODE REVIEW

File: unattended_scripts/wazuh_install.sh [hunk 1]
```diff
@@ -330,9 +330,10 @@ function main() {
             createClusterKey
         fi
         gen_file="${base_path}/certs/password_file.yml"
-        generatePasswordFile 
-        sudo tar -zcf certs.tar -C certs/ .
-        rm -rf "${base_path}/certs"
+        generatePasswordFile
+        eval "tar -zcf '${base_path}/certs.tar' -C '${base_path}/certs/' ."
```
Comment (alberpilot) [inline unattended_scripts/wazuh_install.sh:334]: Add debug variable to end of `eval`

**Review by alberpilot — CHANGES_REQUESTED** [review]
Please review the requested changes

File: unattended_scripts/wazuh_install.sh (hunk 1 above)
Comment (lastranikos) [reply unattended_scripts/wazuh_install.sh:334]: Done!

File: unattended_scripts/wazuh_install.sh [hunk 2]
```diff
@@ -330,9 +330,10 @@ function main() {
             createClusterKey
         fi
         gen_file="${base_path}/certs/password_file.yml"
-        generatePasswordFile 
-        sudo tar -zcf certs.tar -C certs/ .
-        rm -rf "${base_path}/certs"
+        generatePasswordFile
+        eval "tar -zcf '${base_path}/certs.tar' -C '${base_path}/certs/' . $debug""
```
Comment (alberpilot) [inline unattended_scripts/wazuh_install.sh:334]: There are double `"` at the end. Please fix it. 
All variables starting with `$` must use the standard `${variable}`. Please change it with `debug` variable and all others inside `wazuh_install.sh`

**Review by alberpilot — CHANGES_REQUESTED** [review]
Please review the requested changes

File: unattended_scripts/wazuh_install.sh (hunk 2 above)
Comment (lastranikos) [reply unattended_scripts/wazuh_install.sh]: Done!

**Review by alberpilot — APPROVED** [review]

**Pull request merged**